ts/Studio.ts

```diff
@@ -1305,6 +1318,162 @@ export class Studio {
         }
     }
 
+    private bulkFrames(table: HTMLTableElement) {
+        this.setTitle('Bulk Frames');
+
+        let thead = document.createElement('thead');
+        table.appendChild(thead);
+        let tr = document.createElement('tr');
+        thead.appendChild(tr);
+        for (const columnHeader of ['Action', 'Input']) {
+            const th = document.createElement('th');
+            th.textContent = columnHeader;
+            tr.appendChild(th);
+        }
+        const tbody = document.createElement('tbody');
+        table.appendChild(tbody);
+
+        // Move Frame
+        tr = document.createElement('tr');
+        tbody.appendChild(tr);
+        let td = document.createElement('td');
+        td.textContent = 'Move Frame';
+        tr.appendChild(td);
+        td = document.createElement('td');
+        let table2 = document.createElement('table');
+        table2.classList.add('table', 'table-borderless', 'mb-0');
+        td.appendChild(table2);
+        tr.appendChild(td);
+
+        // Move Frame Actions
+        const liftFrames = (value: number) => {
+            this.railroad.frames.forEach((frame: Frame) => {
+                frame.location.z += value;
+            });
+            return value;
+        };
+
+        thead = document.createElement('thead');
+        table2.appendChild(thead);
+        tr = document.createElement('tr');
+        thead.appendChild(tr);
+        for (const columnHeader of ['Axis', 'Value']) {
+            const th = document.createElement('th');
+            th.textContent = columnHeader;
+            tr.appendChild(th);
+        }
+        let tbody2 = document.createElement('tbody');
+        table2.appendChild(tbody2);
+        tr = document.createElement('tr');
+        tbody2.appendChild(tr);
+        td = document.createElement('td');
+        td.textContent = 'Z';
+        tr.appendChild(td);
+        td = document.createElement('td');
+        tr.appendChild(td);
+        const btnLiftAll = document.createElement('button');
+        btnLiftAll.classList.add('btn', 'btn-secondary');
+        btnLiftAll.textContent = 'Lift 5m';
+        btnLiftAll.addEventListener('click', () => {
+            liftFrames(5);
```

Comment:
This is 5cm, not 5m.

```suggestion
            liftFrames(500);
```